src/dsync/dsync.c

```diff
@@ -1071,6 +1502,18 @@ static int dsync_strmap_compare(mfu_flist src_list,
      * list to track files to be deleted from destination */
     mfu_flist src_cp_list     = mfu_flist_subset(src_list);
     mfu_flist dst_remove_list = mfu_flist_subset(dst_list);
+    mfu_flist src_ignore_list = NULL;
```

Comment:
Let's add some comments to describe the purpose of each of these new lists.

Reply:
Done, Added comment to each list.